<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStatusMessageFilterBar</class>
 <widget class="QFrame" name="CStatusMessageFilterBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>58</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Status message filter</string>
  </property>
  <layout class="QGridLayout" name="gl_StatusMessageFilter">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="QFrame" name="fr_Severity">
     <layout class="QHBoxLayout" name="hl_Severity">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_Info">
        <property name="text">
         <string>info</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_Warning">
        <property name="text">
         <string>warning</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_Error">
        <property name="text">
         <string>error</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="swift::gui::filters::CFilterBarButtons" name="filter_Buttons">
     <property name="minimumSize">
      <size>
       <width>75</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="le_Category">
     <property name="placeholderText">
      <string>category</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLineEdit" name="le_MessageText">
     <property name="placeholderText">
      <string>message text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::filters::CFilterBarButtons</class>
   <extends>QFrame</extends>
   <header>gui/filters/filterbarbuttons.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
